styles.css

```diff
@@ -627,7 +628,8 @@ h1 {
   font-family: 'Poppings', sans-serif;
   font-weight: 500;
   font-style: normal;
-  line-height: 16px;font-size: 12px;
+  line-height: 16px;
+  font-size: 12px;
   color: #6070ff;
   background: #ebebff;
   border: none;
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design

  - Your project has a horizontal scroll which shows that your project is not responsive.
  - Your project cards do not have white background color as in the Figma design.
  - Your section design does not look exactly like the Figma design.
  
[Screencast from 02-18-2023 06:04:02 PM.webm](https://user-images.githubusercontent.com/92303891/219881664-5769e49a-ea7d-4344-b882-b08df103daa5.webm)